index.src.html

```diff
@@ -345,15 +345,232 @@
+        <p>
+          A <a>depth map</a> has an associated <dfn>transformation from depth
+          to video</dfn>, which is <a>DepthToVideoTransform</a> dictionary. It
+          represents <dfn>transformation matrix</dfn> of depth camera 3D
+          coordinate system to RGB video stream's camera (identified by
+          <dfn>videoDeviceId</dfn>) 3D coordinate system. After projecting
+          depth 2D pixel coordinates to 3D space, we use this to translate
+          depth camera 3D space coordinates to RGB video camera 3D space.
+        </p>
+        <p>
+          Both depth and color cameras usually introduce significant distortion
+          caused by the camera and lens. While in some cases, the effects are
+          not noticeable, these distortions cause errors in image analysis. To
+          map <a>depth map</a> pixel values to a corresponding color video
+          track pixels, we use two <a>DistortionCoefficients</a> dictionaries:
+          <dfn>deprojection distortion coefficients</dfn> and <dfn>projection
+          distortion coefficients</dfn>.
+        </p>
+        <p>
+          Deprojection coefficients are used for compensating camera distortion
+          when deprojecting 2D pixel coordinates to 3D space coordinates.
+          Projection coefficients are used in the opposite case, when
+          projecting camera 3D space points to pixels. One track doesn't have
+          both of the coefficients specified. The most common scenario is that
+          the depth track has deprojection coefficients or that the color video
+          track has projection coefficients.
+        </p>
+        <p>
+          With this case in mind, the algorithm for mapping depth pixels to
+          color pixels is as follows:
+        </p>
+        <ol>
+          <li>Deproject <a>depth map</a> values to points in depth 3D space.
+            <p>
+              Let <var>dx</var> and <var>dy</var> be 2D coordinates, in pixels,
+              of a pixel in <a>depth map</a>.
+            </p><var></var>
+            <p>
+              Let <var>dz</var> be <a>depth map value</a> of the same pixel in
+              the <a>depth map</a>.
+            </p>
+            <p>
+              Let <var>fx</var> and <var>fy</var> be <a>depth map</a>'s
+              horizontal and vertical <a>focal length</a>s.
+            </p>
+            <p>
+              Let <var>cx</var> and <var>cy</var> be <a>depth map</a>'s
+              <a>principal point</a> 2D coordinates.
+            </p>
+            <p>
+              Let 3D coordinates (Xd, Yd, Zd) be the output of this step - a 3D
+              point in depth camera's 3D coordinate system.
+            </p>
+            <p>
+              `px = (dx - cx) / (fx)`
+            </p>
+            <p>
+              `py = (dy - cy) / (fy)`
+            </p>
+            <ul>
+              <li>If <a>depth map</a>'s <a>deprojection distortion
+              coefficients</a> are not available,
```

Comment:
i use omitted for undefined and defined instead of available now.

in JS, "defined" would be  `if(track.getSettings().deprojectionDistortionCoefficients)   ` - added this note as inline example.